main/lsp/LSPTypechecker.cc

```diff
@@ -450,6 +450,13 @@ bool LSPTypechecker::runSlowPath(LSPFileUpdates updates, WorkerPool &workers, bo
             return;
         }
 
+        // Test-only hook: Stall for as long as `slowPathBlocked` is set.
```

Comment:
WDYT about moving this below the `ENFORCE(gs->lspQuery.isEmpty());` so it's sort of visually obvious that `slowPathBlocked` and `sleepInSlowPath` are doing the same-ish thing?

Reply:
Good call, will do